### PR TITLE
Added Silicon Valley

### PR DESCRIPTION
**Pull request by kunal1244**
- Type of change - Added a new TV series(Silicon Valley).
- The bb_scraper and totl_scraper are posing issues while compiling the index.rb file (although the fix-scraper branch has the corrected code). Also, the entire readme code does not work, because of the Gemfile lock issues, will work on that now.

### CODE REVIEW

File: auto/data/index.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
     "filename": "flash",
-    "imdb_rating": "8.2",
+    "imdb_rating": "7.9",
     "imdb_link": "http://www.imdb.com/title/tt3107288/",
     "wiki_link": "https://en.wikipedia.org/wiki/The_Flash_%282014_TV_series%29",
     "scrape_link": "https://en.wikipedia.org/wiki/List_of_The_Flash_episodes",
-    "seasons": "3",
-    "episodes": "56",
+    "seasons": "5",
+    "episodes": "46",
```
Comment (athityakumar) [inline auto/data/index.json]: Wait, this seems fishy 🤔

File: auto/data/index.json [hunk 2]
```diff
@@ -39,24 +39,24 @@
...
     "filename": "gotham",
-    "imdb_rating": "8.0",
+    "imdb_rating": "7.9",
     "imdb_link": "http://www.imdb.com/title/tt3749900/",
     "wiki_link": "https://en.wikipedia.org/wiki/Gotham_(TV_series)",
     "scrape_link": "https://en.wikipedia.org/wiki/List_of_Gotham_episodes",
-    "seasons": "3",
-    "episodes": "57",
+    "seasons": "5",
+    "episodes": "22",
```
Comment (athityakumar) [inline auto/data/index.json]: Again, number of episodes somehow decreased? 🤔

File: auto/data/index.json [hunk 3]
```diff
@@ -85,14 +85,14 @@
...
     "filename": "lot",
-    "imdb_rating": "7.1",
+    "imdb_rating": "6.9",
     "imdb_link": "http://www.imdb.com/title/tt4532368/",
     "wiki_link": "https://en.wikipedia.org/wiki/Legends_of_Tomorrow",
     "scrape_link": "https://en.wikipedia.org/wiki/Legends_of_Tomorrow",
-    "seasons": "2",
-    "episodes": "26",
+    "seasons": "4",
+    "episodes": "1",
```
Comment (athityakumar) [inline auto/data/index.json]: Same issue as ^^

**Review by athityakumar — COMMENTED** [review]
Hey @kunal1244 👋 

This is such a great start - thanks a lot for fixing the long-awaited scraper bugs. And ofcourse, thanks for adding Silicon Valley scraper as well!

Can you please have a look at the comments I've written (mostly regarding the number of episodes reducing due to undetected episodes)?

**Comment by kunal1244** [comment]
Does seem fishy, didn't notice this beforehand. I didn't change the above scrapers in any way, so these issues shouldn't have arisen. Will have a look.

**Comment by kunal1244** [comment]
Fixed all other faulty scrapers. Works well now.

**Pull request merged**